lxd/network/openvswitch/ovn.go

```diff
@@ -1195,6 +1195,26 @@ func (o *OVN) LogicalSwitchPortAdd(switchName OVNSwitch, portName OVNSwitchPort,
 	return nil
 }
 
+// LogicalSwitchPortIPs returns a list of IPs for a switch port.
+func (o *OVN) LogicalSwitchPortIPs(portName OVNSwitchPort) ([]net.IP, error) {
+	addressesRaw, err := o.nbctl("--format=table", "--no-headings", "--data=bare", "--column=addresses,dynamic_addresses", "find", "logical_switch_port", fmt.Sprintf("name=%s", string(portName)))
```

Comment:
I would encourage you to use the csv format for consistency with other commands, it also helps avoid any misunderstanding of spaces. I did use table in the past and found it had problems parsing it in some cases so generally use csv where possible.

Reply:
Ok, this is what I thought. I will change it.